code/modules/uplink/uplink_items.dm

```diff
@@ -399,7 +399,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	name = "Holoparasites"
 	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
 			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
-	item = /obj/item/storage/box/syndie_kit/guardian
+	item = /obj/item/guardiancreator/tech/rare
```

Comment:
why is this `/rare`

Reply:
its the 15 points that hippiestation gets (the rare holoparasite injector) which only gives more points.

Reply:
First of all, that sentence doesn't even make sense.
Second of all, `/rare` let you choose abilities like timestop and absolution.

Reply:
no it dont because code monkey dumb dumb